client.go

```diff
@@ -101,6 +101,15 @@ type Config struct {
 	// Defaults to 1 second.
 	FetchPollInterval time.Duration
 
+	// ID is the unique identifier for this client. If not set, a random ULID will
```

Comment:
What do you think about saying "random identifier" instead of "random ULID" here?

I'm kind of thinking that we shouldn't commit to the ULID algorithm specifically — it'd be nice to get rid of that dependency, and also I think a more human friendly combination of say host name plus start time might be a better default for operations anyway. Not something we need to do right now, but if we changed it, it'd a minor alteration of the contract above.

Reply:
I made that change and also relocated some relevant language that we had stashed on `JobRow.AttemptedBy` so that the full explanation is only given on the `Config.ID` field now.